src/java/arjdbc/util/ActiveRecord.java

```diff
@@ -0,0 +1,36 @@
+package arjdbc.util;
+
+import org.jruby.RubyClass;
+import org.jruby.RubyModule;
+import org.jruby.runtime.ThreadContext;
+
+/**
+ * Provide a standard place to get references to ActiveRecord classes and modules
+ */
+public class ActiveRecord {
```

Comment:
Not sure on the naming of this one. I kind of like it because now you can call `ActiveRecord.Base(context)` and it mimics the ruby namespacing. I could see it being confusing though and it kind of goes against Java coding conventions.